lib/brcobranca/calculo.rb

```diff
@@ -41,6 +41,48 @@ def modulo11_2to9
       [0, 10, 11].include?(valor) ? 1 : valor
     end
 
+    # Calcula módulo 11 com multiplicadores de 2 a 9 e 2 a 5 (Utilizado pelo Santander).
+    #
+    # @return [Integer]
+    def modulo11_santander
+      return 0 if self.to_i == 0
+
+      somatorio = 0
+      multiplicadores = [2, 3, 4, 5, 6, 7, 8, 9, 2, 3, 4, 5]
+      base = "#{self.rjust(12, '0')}".reverse
+      index = 0
+
+      base.each_char do |char|
```

Comment:
Unused block argument - `char`. You can omit the argument if you don't care about it.